src/worker.js

```diff
@@ -8,11 +8,16 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
+const healthCheckEndpoint = '/';
```

Comment:
+1, would also be nice to be able to specify, the same way the '/batch' endpoint is configurable